x-pack/plugins/fleet/server/services/agents/action_status.ts

```diff
@@ -218,3 +223,11 @@ async function _getActions(
     }, {} as { [k: string]: ActionStatus })
   );
 }
+
+export const rolloutPeriodHasPassed = (source: FleetServerAgentAction) =>
```

Comment:
(nit) This should conventionally be named `hasRolloutPeriodPassed`

```suggestion
export const hasRolloutPeriodPassed = (source: FleetServerAgentAction) =>
```